docs/starters.yml

```diff
@@ -7393,6 +7393,25 @@
     - Uses SCSS for easy-to-understand naming
     - Google Analytics support
     - Responsive design
+- url: https://flat-magazine.netlify.app/
+  repo: https://github.com/damnitrahul/gatsby-starter-flat-magazine
+  description: A flat design Gatsby Starter Blog, Powered by Sanity to manage the content and styled using styled-components.
+  tags:
+    - Blog
+    - CMS:sanity.io
+    - SEO
+    - Search
+    - Pagination
+    - JavaScript
```

Comment:
JavaScript tag isn't valid for starters, only for sites categories in [`categories.yml`](https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml)

Reply:
Oops, Removed it!